src/Covid19Dashboard/index.jsx

```diff
@@ -113,9 +113,20 @@ class Covid19Dashboard extends React.Component {
           <XAxis
             dataKey='date'
             tick={<CustomizedXAxisTick />}
-            interval={1}
+            interval={7}
           />
           <YAxis
+            label={{ value: 'confirmed/recovered', angle: -90, position: 'insideLeft' }}
```

Comment:
could you have a look at having it just say "confirmed" when the chart doesn't include recovered counts?

Reply:
updated